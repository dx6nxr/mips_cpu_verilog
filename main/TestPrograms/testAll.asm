lui $2, 0x0000
ori $2, 493
lui $3, 0x0000
ori $3, 23
bltz $2, inc
jal mult

main:
mfhi $2
mflo $3
j exit


inc: 
addu $2, $2, $2
j main

mult:
multu $2, $3
jr $ra
exit:
