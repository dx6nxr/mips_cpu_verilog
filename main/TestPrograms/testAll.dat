0x3c020000
0x344201ed
0x3c030000
0x34630017
0x04400004
0x0c10000b
0x00001010
0x00001812
0x0810000d
0x00421021
0x08100006
0x00430019
0x03e00008
