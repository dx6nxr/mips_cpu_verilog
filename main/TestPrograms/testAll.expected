@3 2c4b
@1f 400018